src/components/atoms/UserAvatar/UserAvatar.scss

```diff
@@ -65,10 +76,28 @@
     }
   }
 
-  @each $modifier, $size in $avatar-sizes-map {
+  @each $modifier, $avatar-size, $indicator-size,
+    $indicator-border-width in $avatar-sizes-enum
+  {
     &--#{$modifier},
     &--#{$modifier} img {
-      @include square-size($size);
+      @include square-size($avatar-size);
+    }
+
+    &--#{$modifier} &__status-indicator {
+      @include square-size($indicator-size);
+      border-width: $indicator-border-width;
+
+      // NOTE: variable is just template for DRY, the actual calc() is performed in the browser
+      // calculated out of both circles sizes and offset for the bottom/right side of the indicator
+      $indicator-position-offset: calc(
+        #{$avatar-size} *
+          #{$indicator-offset-factor} -
+          #{$indicator-size} *
+          #{$indicator-radius-factor}
+      );
```

Comment:
SCSS is a templating engine, so in this case it will not pre-calculate the value but pass it to the browser to use the `calc()` function on the substituted values.

This is used because browsers have more flexibility when it comes to mixing calculations of pixels and percents that aren't available to the templating engine  during compile time.